codec-http/src/main/java/io/netty/handler/codec/http/CombinedHttpHeaders.java

```diff
@@ -92,9 +93,7 @@ public CharSequence escape(CharSequence value) {
                 return itr;
             }
             Iterator<CharSequence> unescapedItr = unescapeCsvFields(itr.next()).iterator();
-            if (itr.hasNext()) {
-                throw new IllegalStateException("CombinedHttpHeaders should only have one value");
-            }
+            checkState(!itr.hasNext(), "CombinedHttpHeaders should only have one value");
```

Comment:
Tbh checkState seems ambiguous, assertTrue? assertFalse?

Reply:
i  will check, do you have a correct codestyle.xml ? @johnou

Reply:
https://github.com/netty/netty-build/blob/master/src/main/resources/io/netty/checkstyle.xml

Reply:
Not this one, i said it's codeStyle.xml, i downloaded from netty-website,imported to idea IDE. But when i reformat code,  i found it's not correct. I confused.